Sources/Geocodable.swift

```diff
@@ -22,52 +22,81 @@ public protocol Geocodable {
     func geocodeLocality(_ string: String,
                          forCountry country: String,
                          completion: @escaping (Result<GeocodedAddress, GeocodingError>) -> ())
-
-}
-
-public extension Geocodable {
-
-    /// Geocodes the given string into a geocoded address.
+    
+    /// Geocodes the given string into a geocoded location.
     ///
     /// - Parameters:
     ///   - string: The string to geocode.
     ///   - country: The country to use as a filter. Default value is "US".
     ///   - completion: The completion block.
     func geocodeLocality(_ string: String,
```

Comment:
@ThibaultKlein Wondering if the function name here makes sense anymore. Maybe they should be `geocodeAddress(fromZIP: forCountry: completion:)` and `geocodeLocation(fromZIP: forCountry: completion:)` now instead? What do you think?